bigchaindb-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,246 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "admin_username": {
+            "defaultValue": null,
+            "type": "String",
+            "metadata": {
+                "description": "The admin username"
+            }
+        },
+        "admin_password": {
+            "defaultValue": null,
+            "type": "SecureString",
+            "metadata": {
+                "description": "The admin password"
+            }
+        },
+        "virtual_machine_name": {
+            "defaultValue": "",
+            "type": "String",
+            "metadata": {
+                "description": "A name for the virtual machine"
+            }
+        },
+        "virtual_machine_size": {
+            "defaultValue": "Standard_D2",
+            "type": "String",
+            "metadata": {
+                "description": "The virtual machine size"
+            },
+            "allowedValues": [

```

Comment:
remove allowed values, it will prevent deployment on other cloud/regions that don't support these sizes